node/docs/genmd.js

```diff
@@ -102,6 +102,8 @@ writeLine();
 writeLine('The build generates a vsts-task-lib.d.ts file for use when compiling tasks');
 writeLine('In the example below, it is in a folder named definitions above the tasks lib');
 writeLine();
+writeLine('## [Release notes](releases.md)');
```

Comment:
I'm unable to validate this change. I get errors on Windows when attempting to run docs/genmd.js directly or when attempting to run docs.sh.
```
Error: C:/vsts-task-lib/node/typings/main/ambient/node/index.d.ts(84)
 Type expected.
Error: C:/vsts-task-lib/node/typings/main/ambient/node/index.d.ts(187)
 Type expected.
Error: C:/vsts-task-lib/node/typings/main/ambient/node/index.d.ts(188)
 Type expected.
[omitted about 20 more]
```